examples/X_NUCLEO_IHM02A1_HelloWorld/X_NUCLEO_IHM02A1_HelloWorld.ino

```diff
@@ -139,6 +139,7 @@ void setup()
 {
     /*----- Initialization. -----*/
 
+    /* Please verify your board has these spi pins */
     /* Initializing SPI bus. */
     dev_spi = new SPIClass(D11, D12, D3);
```

Comment:
I think issue is more on this line, probably a mistyping.
This should be per default:
`  dev_spi = new SPIClass(D11, D12, D13);`
@cparata are you agreed?

Reply:
It's probably intended like this, since the default soldering bridge configuration leads to D3 being SCK.

Reply:
Right, just checked the schematics... Sorry.
So yes, it should be fine to warn end user.
But I guess, it should be also in the comment header as done in several Arduino examples.

Reply:
Ok. It is fine for me. I will merge this comment.